AjobK/Athena
# Added folder structure, Also added some files

# Athena Back-end Setup

## Changes

- Added for back-end folders.

- Added MongoDB

- Added Express

### How to test:

- Pull from ```#9-backend-folder-structure``` check for linting errors & for clean code.

#### Note
```(node:5532) DeprecationWarning: current URL string parser is deprecated, and will be removed in a future version. To use the new parser, pass option { useNewUrlParser: true } to MongoClient.connect.```
